addons/im_livechat/controllers/main.py

```diff
@@ -195,14 +195,14 @@ def get_session(self, channel_id, anonymous_name, previous_operator_id=None, cha
 
     def _post_feedback_message(self, channel, rating, reason):
         reason = Markup("<br>" + re.sub(r'\r\n|\r|\n', "<br>", reason) if reason else "")
-        body = Markup('''
-            <div class="o_mail_notification o_hide_author">
-                %(rating)s: <img class="o_livechat_emoji_rating" src="%(rating_url)s" alt="rating"/>%(reason)s
-            </div>
-        ''') % {
-            'rating': _('Rating'),
-            'rating_url': rating.rating_image_url,
-            'reason': reason,
+        body = Markup(
+            """<div class="o_mail_notification o_hide_author">"""
```

Comment:
Note: only "cosmetic" change, since we now assert this value in test, it was obvious how ugly it was (many new lines and a lot of unnecessary space in body), so the opportunity is taken to simplify it.